### ISSUE TITLE
AST: Inconsistent casing of location in serialized comment nodes

### ISSUE THREAD

**Issue by anderseknert** [open-policy-agent/opa#6258]
When serialized, comment nodes display the location and text attributes as `Location` and `Text`. This is inconsistent with how those attributes are presented anywhere else in the AST (where they are lowercased), and it would be good to have that fixed.

```rego
package p

# foo bar
```
```shell
opa parse --format json p.rego
```
```json
{
  "package": {
    "path": [
      {
        "type": "var",
        "value": "data"
      },
      {
        "type": "string",
        "value": "p"
      }
    ]
  },
  "comments": [
    {
      "Text": "IGZvbyBiYXI=",
      "Location": {
        "file": "c.rego",
        "row": 3,
        "col": 1
      }
    }
  ]
}
```

**Comment by prashant3286** [comment]
Hello @anderseknert . I would like to contribute to this bug, can you please assigned this issue to me?

**Comment by anderseknert** [comment]
Hi @prashant3286! And thank you for offering to help! The issues that are tagged `1.0` are a little special compared to other issues, as they are breaking changes that can't be included in our monthly releases. That means special care will need to be taken either to guard these features behind feature flags, or simply delay them until the 1.0 release. In this case, I'm not sure it'd be feasible to change serialization format based on a feature flag, so I'll defer this to @johanfylling and @ashutosh-narkar, who are leading the 1.0 effort.

**Comment by prashant3286** [comment]
Sure. Please let me know if the breaking changes will be implemented soon so that I could contribute to the other issue.
Also please suggest me some bug which needs to be fixed asap.

**Comment by anderseknert** [comment]
Starting out with one of the issues marked as "[good first issue](https://github.com/open-policy-agent/opa/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)" is a good way to get your feet wet before going after bigger fish :)

**Comment by anderseknert** [comment]
Not much point in keeping this open since it isn't included for 1.0. Closing.
